formbricks/formbricks
# fix: fixed testimonial image overflowing from it's container

## What does this PR do?
it resolves the issue of a image overflowing from it's container

Fixes # (issue)
From this
![image](https://github.com/formbricks/formbricks/assets/66205352/a2d9db88-dc93-4485-a0ff-0538a9d8ea1b)
to this
![image](https://github.com/formbricks/formbricks/assets/66205352/96e74b63-02d5-4029-9a15-378e786c0862)

## How should this be tested?
Test using the developer tools and select various devices on it to check it out.

## Checklist

### Required


### Appreciated
